### ISSUE TITLE
Error while using ptr dl_docs

### ISSUE THREAD

**Issue by HiidanX** [pytr-org/pytr#151]
**Description of the bug**
I have been getting an error for two weeks running pytr dl_docs.

**To Reproduce**

2. Ran the command `pytr dl_docs`

**Expected behavior**
Download of all pdf files from Trade Republic

**Error log**

/.local/pipx/venvs/pytr/lib/python3.12/site-packages/pytr/timeline.py", line 130, in process_timelineDetail
    event = self.timeline_events[response["id"]]
KeyError: '05fcd862-75f4-4271-8236-60ebf207b2f2'

**Environment**

- OS: Mac
- pytr version: 0.3.1

**Additional context**
This used to work for me without a problem. Now there seems to be a problem. Anyone else noticed similar behaviour?
Root Cause: Christmas present

**Comment by HiidanX** [comment]
I have narrowed down the problem. When including the parameter last_days parameter it seems to work. There seems to be problem when exporting the christmas present (Weihnachtsgeschenk). Is there a way to fix this?

**Comment by peterandree** [comment]
doesn't work for me even with --last days

**Comment by peterandree** [comment]
> doesn't work for me even with --last days

have to correct myself. When I triangulate last days so, that the christmas gift is not in the timespan, it seems to work. Good to know, but somewhat useless since I need to download older values too

**Comment by HiidanX** [comment]
Yes, a bugfix is needed to somehow correct the christmas present. Has anyone solved the problem already?

**Comment by HiidanX** [comment]
It seems to the same error as the issue #148

**Comment by otmezger** [comment]
In my case, it seems to be breaking with a `stock gift`. I had 2 of them in December.

**Comment by NiklasRosenstein** [comment]
Closing as duplicate of #148 (and that issue was addressed).
